consensus/istanbul/core/commit.go

```diff
@@ -30,15 +31,43 @@ func (c *core) sendCommit() {
 	c.broadcastCommit(sub)
 }
 
-func (c *core) generateCommittedSeal(sub *istanbul.Subject) ([]byte, error) {
+func (c *core) generateCommittedSeal(sub *istanbul.Subject) (blscrypto.SerializedSignature, error) {
 	seal := PrepareCommittedSeal(sub.Digest, sub.View.Round)
 	committedSeal, err := c.backend.SignBlockHeader(seal)
 	if err != nil {
-		return nil, err
+		return blscrypto.SerializedSignature{}, err
 	}
 	return committedSeal, nil
 }
 
+func (c *core) generateEpochValidatorSetSeal() (blscrypto.SerializedSignature, error) {
```

Comment:
We need a `verifyEpochValidatorSetSeal` just like `verifyCommittedSeal`. Otherwise, a single validator would be able to sabotage consensus by sending a commit with an invalid `EpochValidatorSetSeal`, which would be admitted and cause `GetAggregatedEpochSeal` to fail.

Reply:
I'm pretty sure we had a similar discussion about it in the past, where we also added this to normal signatures. 
I agree we should add it, but I don't think it would fail right now more than it would fail with the added check, since what we do now is basically just add points.

Reply:
I'm not sure I understand

Reply:
FYI, I already added these checks!
I can see some problematic scenarios without them.